Components/avatar/avatar.css

```diff
@@ -0,0 +1,24 @@
+
+.avatar {
+    width: 3rem;
+}
```

Comment:
I think It'll be better if we provide it with a fixed height too.
Otherwise, the avatar's height might stretch

Reply:
Thanks Shobit, I'll look into it and make the required changes!

Reply:
if we don't use height then it will take default as "auto" which is fine for avatar.